bin/picca_deltas.py

```diff
@@ -400,17 +402,19 @@ def cont_fit(data):
     st = interp1d(ll_st[wst>0.],st[wst>0.],kind="nearest",fill_value="extrapolate")
     deltas = {}
     data_bad_cont = []
-    for p in sorted(list(data.keys())):
+    for p in sorted(data.keys()):
         deltas[p] = [delta.from_forest(d,st,forest.var_lss,forest.eta,forest.fudge, args.use_mock_continuum) for d in data[p] if d.bad_cont is None]
         data_bad_cont = data_bad_cont + [d for d in data[p] if d.bad_cont is not None]
 
     for d in data_bad_cont:
-        log.write("rejected {} due to {}\n".format(d.thid,d.bad_cont))
+        log.write("INFO: Rejected {} due to {}\n".format(d.thid,d.bad_cont))
+
+    log.write("INFO: Sample has {} forests\n".format(sp.sum([len(p) for p in deltas.values()])))
```

Comment:
see above

Reply:
Good idea, let's keep that in mind for when we have the time.